src/lib/init.ts

```diff
@@ -376,14 +375,38 @@ async function initTSorJSGatewayProject({
     },
   };
 
+  await ensureDir(join(getGatewaySrcPath(), "app"));
+  await ensureDir(join(getGatewayPath(), "api"));
+  await ensureDir(join(getGatewayPath(), "public"));
+
   await Promise.all([
     fluenceConfig.$commit(),
-    writeFile(indexFilePath, getGatewayIndexJsContent(isJS), FS_OPTIONS),
+    writeFile(
+      join(getGatewayPath(), "src", "app", `index.${isJS ? "js" : "ts"}`),
```

Comment:
a bit cleaner to get the estension once and then use it in all of the cases where it is needed